util/pbtypes/copy.go

```diff
@@ -158,3 +161,24 @@ func CopyFilter(in *model.BlockContentDataviewFilter) (out *model.BlockContentDa
 	bytesPool.Put(buf)
 	return
 }
+
+func CopyNotification(in *model.Notification) (out *model.Notification) {
```

Comment:
dunno if it is possible to have a generic version of the method, don't do if too tedious